src/test/java/org/openmrs/performance/registries/DoctorRegistry.java

```diff
@@ -132,6 +132,8 @@ public ChainBuilder openAttachmentsTab(String patientUuid) {
 				.exec(httpService.getAllowedFileExtensions());
 	}
 	
+	public ChainBuilder openProgramsTab(String patientUuid) {return exec(httpService.getPrograms(patientUuid));}
```

Comment:
The formatting is a bit weird here, try adding the openmrs formatter profile to your IDE. 
https://openmrs.atlassian.net/wiki/spaces/docs/pages/25469726/Developer+How-To+Setup+And+Use+Eclipse#%5BinlineExtension%5DJava-Code-Format

Reply:
Sure i'll do that.